packages/neos-ui/src/manifest.js

```diff
@@ -21,6 +15,13 @@ manifest('main', {}, globalRegistry => {
     `));
 
     //
+    // Create inline editor registry
+    //
+    globalRegistry.add('inlineEditors', new SynchronousRegistry(`
+        # Registry for inline editors
```

Comment:
documentation here is required; explaining the basic API here and interacting registries